test/metabase/models/field_values_test.clj

```diff
@@ -123,6 +123,18 @@
   (sync/sync-database! db)
   (find-values field-values-id))
 
+(deftest get-or-create-full-field-values!-test
+  (testing "create a full Fieldvalues if it does not exist"
+    (db/delete! FieldValues :field_id (mt/id :categories :name) :type :full)
+    (is (= :full (:type (field-values/get-or-create-full-field-values! (Field (mt/id :categories :name)))))
```

Comment:
This might be clearer as a `->`? That might just be my tastes.

Reply:
yes, it does look more readable. I'll change it to use threading